decktape.js

```diff
@@ -275,274 +279,275 @@ process.on('unhandledRejection', error => {
       .then(_ => exportSlides(plugin, page, pdf))
       .then(async context => {
         await writePdf(options.filename, pdf);
-        console.log(chalk`{green \nPrinted {bold ${context.exportedSlides}} slides}`);
+        console.log(chalkTemplate`{green \nPrinted {bold ${context.exportedSlides}} slides}`);
         browser.close();
         process.exit();
       }))
     .catch(error => {
-      console.log(chalk`{red \n${error}}`);
+      console.log(chalkTemplate`{red \n${error}}`);
       browser.close();
       process.exit(1);
     });
 
-})();
 
-async function writePdf(filename, pdf) {
-  const pdfDir = path.dirname(filename);
-  try {
-    fs.accessSync(pdfDir, fs.constants.F_OK);
-  } catch {
-    fs.mkdirSync(pdfDir, { recursive: true });
+  async function writePdf(filename, pdf) {
```

Comment:
Moved all these functions inside the anonymous `async` function, because most of them seem to use the `options` that are now parsed at the top of the async function. (no longer a global variable)